### PR TITLE
[vcpkg-ci-ffmpeg] remove tensorflow feature from dependencies

### PR DESCRIPTION
**Pull request by strega-nil-ms**
This means that ffmpeg _actually gets built_

### CODE REVIEW

**Comment by Neumann-A** [comment]
Hmm I would rather like to see `vcpkg-ci-ffmpeg` to be modified instead of the `ffmpeg` port.

File: scripts/test_ports/vcpkg-ci-ffmpeg/vcpkg.json [hunk 1]
```diff
@@ -7,9 +7,143 @@
   "dependencies": [
     {
       "name": "ffmpeg",
+      "default-features": false
+    },
```
Comment (ras0219-msft) [inline scripts/test_ports/vcpkg-ci-ffmpeg/vcpkg.json:11]:
```suggestion
```

This is implied by the second dependency item

Comment (ras0219-msft) [inline scripts/test_ports/vcpkg-ci-ffmpeg/vcpkg.json:11]:
```suggestion
```

This is implied by the second dependency item

**Review by ras0219-msft — APPROVED** [review]
LGTM assuming it fixes the problem

**Comment by strega-nil-ms** [comment]
```
vcpkg-ci-ffmpeg:x64-windows: SUCCEEDED: 136.5 ms
```

**Pull request merged**